zmon_cli/client.py

```diff
@@ -63,6 +63,8 @@ def wrapper(*args, **kwargs):
 
 
 def compare_entities(e1, e2):
+    e1.pop('last_modified', None)
```

Comment:
this works, but it modifies the incoming `dict`, i.e. might lead to unexpected and dangerous behavior